### ISSUE TITLE
Slashes are no longer removed from the titles

### ISSUE THREAD

**Issue by vers-one** [lay295/TwitchDownloader#662]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Edition

Windows GUI Application

### Describe your issue here

Previously, VOD titles like `apple/banana/cat` would be transformed to `applebananacat` to be used in the file name (not ideal but manageable). Now, after the change in the PR #638 they are transformed into `apple\banana\cat` which causes the "Path not found" error in the save file dialog, unless the user removes the slashes manually.

I think a better way would be to call `RemoveInvalidChars` on the `title` first, before passing it to the template. Or even better, to replace all invalid characters with `_` to preserve word breaks.

**Comment by ScrubN** [comment]
Good catch, I didn't realize that was a potential issue when merging the PR.

**Comment by ScrubN** [comment]
Fixed eb299c18fdeaa0338abdb18b3b39e930e49e1154
